docker-images/sw360populated/Dockerfile

```
@@ -6,7 +6,7 @@
 # which accompanies this distribution, and is available at
 # http://www.eclipse.org/legal/epl-v10.html
 
-FROM sw360/sw360empty:latest
+FROM sw360_lr7/sw360empty:latest
```

is the variable `$COMPOSE_PROJECT_NAME` available here? Then it might better to write:
```suggestion
FROM ${COMPOSE_PROJECT_NAME}/sw360empty:latest
```